doc/btp_gap.txt

```diff
@@ -510,6 +510,115 @@ Commands and responses:
 
 		In case of an error, the error response will be returned.
 
+
+	Opcode 0x22 - Configure Periodic Advertising command/response
+
+        Controller Index:	<controller id>
+		Command parameters:	 Flags (1 octet)
+		                     Interval Min (2 octets)
+		                     Interval Max (2 octets)
+		Return Parameters:	<none>
+
+        This command is used to configure Periodic Advertising instance.
+        IUT is expected to have 1 Periodic Advertising instance.
+
+		Possible values for the Flags parameter are a bit-wise OR
+		of the following bits:
+					0 = Include TX Power
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x23 - Periodic Advertising Start command/response
```

Comment:
Periodic Advertising Stop command should also be defined (even if currently no test requires it)